Dockerfile

```diff
@@ -1,28 +1,25 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
 
-LABEL authors='Chris Rust <chris.rust@solarwinds.com>, Dawid Śmiech <dawid.smiech@solarwinds.com>'
-
-ARG swisnap_version
-ARG swisnap_repo=swisnap
+LABEL authors='Dawid Śmiech <dawid.smiech@solarwinds.com>'
```

Comment:
Is this still accurate?

Reply:
It's not -- he's no longer with the company. I'm not sure we need an author label for this tbh.